hedera-mirror-common/src/main/java/com/hedera/mirror/common/aggregator/LogsBloomAggregator.java

```diff
@@ -0,0 +1,67 @@
+package com.hedera.mirror.common.aggregator;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public class LogsBloomAggregator {
+
+    public static final int BYTE_SIZE = 256;
+    private byte[] aggregatedBlooms = new byte[0];
```

Comment:
Empty byte array is immutable so we can optimize.
```suggestion
    private byte[] aggregatedBlooms = ArrayUtils.EMPTY_BYTE_ARRAY;
```